### PR TITLE
Adding event listener for test updated

### PR DESCRIPTION
**Pull request by Jnegrier**
Maybe we could take advantage of this PR and add the other event? what you think?

### CODE REVIEW

**Comment by opishcheiko** [comment]
@Jnegrier  what do you think is the best? suite status?

**Comment by Jnegrier** [comment]
> @Jnegrier what do you think is the best? suite status?

Yeah, that sounds like the natural new step @opishcheiko 👍

**Comment by opishcheiko** [comment]
@Jnegrier i had to rearrange a bit how we send suites data to the component, but it shouldn't be too painful to review. It seems to work 😉

File: components/templates/ListOfTests.tsx [hunk 1]
```diff
@@ -123,6 +123,19 @@ useSocket('delta_resolution', testResolution => {
     }
 })
 
+useSocket('delta_test', testDelta => {
+  let filteredTest = tests.find(
+    test => test.mother_test_id === testDelta.mother_test_id);
```
Comment (Jnegrier) [inline components/templates/ListOfTests.tsx:128]: Actually, looks like I made a mistake here, this should just `test_id`, as a `mother_test_id` would change the status of a lot of non related test 😧

**Review by opishcheiko — APPROVED** [review]
good that you noticed! Noice

**Pull request merged**